pkg/querier/chunk_store_queryable.go

```diff
@@ -44,10 +44,11 @@ func (q *chunkStoreQuerier) Select(sp *storage.SelectParams, matchers ...*labels
 		return nil, nil, promql.ErrStorage{Err: err}
 	}
 
-	return q.partitionChunks(chunks), nil, nil
+	return partitionChunks(chunks, q.mint, q.maxt, q.chunkIteratorFunc), nil, nil
 }
 
-func (q *chunkStoreQuerier) partitionChunks(chunks []chunk.Chunk) storage.SeriesSet {
+// Series in the returned set are sorted by labels.
```

Comment:
It isn't immediately obvious to me what this means.  Sorted by ASCII ordering of `Labels.String()` ?  Sorted by fingerprint hash?

Reply:
They are sorted alphabetically by labels. This happens inside `newConcreteSeriesSet`, and is a prerequisite for efficient merging by `storage.NewMergeSeriesSet`. I will update comment to be more clear.